### PR TITLE
Update terms

### PR DESCRIPTION
**Pull request by adrianwells**
"Microsoft Pinpoint" appears to be renamed to "Microsoft Solution Providers", at least that is where the URL in this article references. So updating the terms. Also can this document have additional notes about how to search for fax solution providers under "Microsoft Solution Providers" as it does not appear to yield results. Or should another URL be referenced instead?

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/voice-mail-unified-messaging/set-up-client-voice-mail-features/set-up-incoming-faxing.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Before you can set up faxing for your organization, you need to successfully con
 ### Step 2: Configure fax partner servers
 <a name="step2configurefax"> </a>
 
-Next, you need to enable incoming faxing and configure the fax partner's URI on each UM mailbox policy that you require in your organization. To successfully deploy incoming faxing, you must integrate a certified fax partner solution with Exchange Unified Messaging. For details, see [Fax advisor for Exchange UM](fax-advisor-for-exchange-um.md). For a list of certified fax partners, see [Microsoft Pinpoint for Fax Partners](https://go.microsoft.com/fwlink/p/?LinkId=190238)
+Next, you need to enable incoming faxing and configure the fax partner's URI on each UM mailbox policy that you require in your organization. To successfully deploy incoming faxing, you must integrate a certified fax partner solution with Exchange Unified Messaging. For details, see [Fax advisor for Exchange UM](fax-advisor-for-exchange-um.md). For a list of certified fax partners, see [Microsoft Solution Providers for Fax Partners](https://go.microsoft.com/fwlink/p/?LinkId=190238)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/voice-mail-unified-messaging/set-up-client-voice-mail-features/set-up-incoming-faxing.md]:
```suggestion
Next, you need to enable incoming faxing and configure the fax partner's URI on each UM mailbox policy that you require in your organization. To successfully deploy incoming faxing, you must integrate a certified fax partner solution with Exchange Unified Messaging. For details, see [Fax advisor for Exchange UM](fax-advisor-for-exchange-um.md). For a list of certified fax partners, see [Microsoft Solution Providers for Fax Partners](https://go.microsoft.com/fwlink/p/?LinkId=190238).
```

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by mattpennathe3rd** [comment]
@adrianwells Thanks very much for submitting this. I'm checking into your search query question before I merge and will report back soon.

**Comment by mattpennathe3rd** [comment]
@adrianwells It turns out that in Exchange Online, this entire feature set is part of Unified Messaging, which was retired last month. I have a large editing project slated for the latter half of April to remove all references to Unified Messaging throughout the Exchange Online docs (though Exchange Server on-premises is not affected), so all of this will go away in the near future and as such, this PR is sort of moot!

However, we do have a number of references to Pinpoint in the older Exchange 2013 docs, so I will open a new PR to address that. Thanks for helping out!

**Pull request closed without merging**